packages/api/src/controllers/notification.ts

```diff
@@ -0,0 +1,107 @@
+import { sendgridEmail } from "./helpers";
+import { frontendUrl, unsubscribeUrl } from "./user";
+import { db } from "../store";
+import { sendgridEmailPaymentFailed } from "./helpers";
+import { WithID } from "../store/types";
+import { Request } from "express";
+import { User } from "../schema/types";
+
+export const getUsageNotifications = async (
+  usagePercentages,
```

Comment:
Can you add a type to this? Can even inline it e.g.
```suggestion
  usagePercentages: {TotalUsageMins: number, DeliveryUsageMins: number, StorageUsageMins: number},
```